chart/.gitignore

```diff
@@ -0,0 +1,4 @@
+bundles/
+es/
+lib/
+!src/lib
```

Comment:
!src/lib doesn't seem like a good idea ..

Reply:
I want to enable root/src/lib here. But I do want to enble root/lib. 
root/src/lib stores the source code
root/lib.      stores the built version
Do you know any better idea?

Reply:
did you look at the documents https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring ?

Reply:
Thanks for this. Now I use the config in the Momentum Root